content/blog/2022-2-15-february-22-heartbeat.md

```diff
@@ -0,0 +1,367 @@
+---
```

Comment:
@jendefig merged and noticed after ^^ the name should be updated 2022-2-15 - 2022-02-17?

Reply:
Ah.  Crud.  Ok.